layouts/partials/docs/feedback.html

```diff
@@ -0,0 +1,42 @@
+<div id="docsFeedbackContainer" class="mt-8 md:mt-0 hidden">
+    <h4 class="no-anchor text-gray-600 uppercase tracking-wide font-bold text-sm">Feedback</h4>
+    <div id="feedbackButtons">
+        <p class="mt-3 mb-5 text-sm">Was this page helpful?</p>
+        <div class="flex">
+            <a id="docsFeedbackYes" data-track="feedback-yes" class="btn flex">
+                <i class="fas fa-thumbs-up mr-2 text-lg"></i>
+                Yes
+            </a>
+            <a id="docsFeedbackNo" data-track="feedback-no" class="btn flex mx-5">
+                <i class="fas fa-thumbs-down mr-2 text-lg"></i>
+                No
+            </a>
+        </div>
+    </div>
+    <div id="feedbackThankYou" class="hidden">
+        {{ $problemTitle := printf "Issue with %s" .Permalink }}
+        {{ $improvementTitle := printf "Improvement for %s" .Permalink }}
+        <p>
+            Thank you for your feedback. If you have a specific, answerable question about how
+            to use Pulumi, ask it in our <a href="https://slack.pulumi.com/">Community Slack</a>.
+            Open an issue in the GitHub repo if you want to
+            <a href="https://github.com/pulumi/docs/issues/new?title={{ $problemTitle }}">report a problem</a> or
+            <a href="https://github.com/pulumi/docs/issues/new?title={{ $improvementTitle }}">suggest an improvement</a>.
+        </p>
+    </div>
+    <div id="feedbackLongForm" class="hidden">
+        <div class="popup-overlay">
```

Comment:
It looks like there's only a SUBMIT button but not a cancel button, and blur/click-outside doesn't seem to dismiss it. We should handle being able to dismiss the dialog without submitting feedback.

It's also unclear when I click a thumb whether the sentiment was captured or whther I have to take the additional step of clicking the SUBMIT button also. I'd expect to be able to just click +1/-1 and then optionally be invited to submit additional feedback after that. Does that make sense?